keel-core/src/main/kotlin/com/netflix/spinnaker/keel/services/ResourceHistoryService.kt

```diff
@@ -48,22 +45,10 @@ import org.springframework.stereotype.Component
  */
 @Component
 class ResourceHistoryService(
-  private val repository: KeelRepository,
+  private val resourceRepository: ResourceRepository,
   private val actuationPauser: ActuationPauser
 ) {
 
-  /**
-   * Returns the history of events associated with the specified resource from the database, plus any applicable
-   * application-level pause/resume events.
-   */
-  fun getEnrichedEventHistory(id: String, limit: Int = DEFAULT_MAX_EVENTS): List<PersistentEvent> {
```

Comment:
Another function gone in favor of simply querying the right records from the data store.